tests/test_bsyncpy.py

```diff
@@ -49,13 +49,13 @@ def test_datetime():
     """
     Added for https://github.com/BuildingSync/bsyncpy/issues/6
     """
-    dt = datetime.datetime(2019, 1, 1, 0, 0, 0)
-    ts = bsync.TimeSeriesData.TimeSeries.StartTimestamp(dt)
+    dt = datetime.datetime(2019, 1, 1, 8, 24, 59)
+    ts = bsync.StartTimestamp(dt)
```

Comment:
several elements were promoted to complex/simple types, so they are not at the top level.